Makefile fixes.

Index: sysstat/Makefile.in
===================================================================
--- sysstat.orig/Makefile.in	2011-01-19 21:14:31.000000000 +0100
+++ sysstat/Makefile.in	2011-01-19 21:22:40.000000000 +0100
@@ -61,9 +61,9 @@
 SYSCONFIG_DIR = @SYSCONFIG_DIR@
 
 # Compiler flags
-CFLAGS = @CFLAGS@ -Wall -Wstrict-prototypes -pipe -O2
+CFLAGS = @CFLAGS@ -Wall -Wstrict-prototypes -pipe
 DFLAGS = @DFLAGS@
-LFLAGS =  -s
+LFLAGS = @LDFLAGS@
 DFLAGS += -DSA_DIR=\"$(SA_DIR)\" -DSADC_PATH=\"$(SADC_PATH)\"
 
 # Commands
@@ -219,8 +219,9 @@
 locales:
 endif
 
-nls/sysstat.pot: $(wildcard *.c)
-	$(XGETTEXT) -o $@ -k_ --msgid-bugs-address="sysstat <at> orange.fr" $^
+.PHONY: nls/sysstat.pot
+nls/sysstat.pot: | $(wildcard *.c)
+	$(XGETTEXT) -o $@ -k_ --msgid-bugs-address="sysstat <at> orange.fr" $|
 
 # Phony targets
 .PHONY: clean distclean install install_base install_all uninstall \
@@ -342,7 +343,7 @@
 	-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(SA_DIR)/[0-9]?????
 endif
 	mkdir -p $(DESTDIR)$(BIN_DIR)
-	mkdir -p $(DESTDIR)$(DOC_DIR)
+	#mkdir -p $(DESTDIR)$(DOC_DIR)
 	mkdir -p $(DESTDIR)$(SYSCONFIG_DIR)
 	$(INSTALL_BIN) sa1 $(DESTDIR)$(SA_LIB_DIR)
 	$(INSTALL_BIN) sa2 $(DESTDIR)$(SA_LIB_DIR)
@@ -360,12 +361,12 @@
 	$(INSTALL_DATA) sysstat.ioconf $(DESTDIR)$(SYSCONFIG_DIR)
 	$(INSTALL_DATA) sysstat.sysconfig $(DESTDIR)$(SYSCONFIG_DIR)/sysstat
 ifeq ($(INSTALL_DOC),y)
-	$(INSTALL_DATA) CHANGES $(DESTDIR)$(DOC_DIR)
-	$(INSTALL_DATA) COPYING $(DESTDIR)$(DOC_DIR)
-	$(INSTALL_DATA) CREDITS $(DESTDIR)$(DOC_DIR)
-	$(INSTALL_DATA) README $(DESTDIR)$(DOC_DIR)
-	$(INSTALL_DATA) FAQ $(DESTDIR)$(DOC_DIR)
-	$(INSTALL_DATA) *.lsm $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) CHANGES $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) COPYING $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) CREDITS $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) README $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) FAQ $(DESTDIR)$(DOC_DIR)
+#	$(INSTALL_DATA) *.lsm $(DESTDIR)$(DOC_DIR)
 endif
 
 install_all: install_base cron/crontab sysstat \
@@ -578,7 +579,7 @@
 	rm -f sadc sar sadf iostat mpstat pidstat nfsiostat cifsiostat *.o *.a core TAGS
 	find nls -name "*.gmo" -exec rm -f {} \;
 
-almost-distclean: clean nls/sysstat.pot
+almost-distclean: clean
 	rm -f sa1 sa2 sysstat cron/crontab version.h sysconfig.h
 	rm -f sysstat.sysconfig cron/sysstat.crond cron/sysstat.cron.daily
 	rm -f cron/sysstat.cron.hourly cron/sysstat.crond.sample cron/sysstat.crond.sample.in
