Minor fixes in config files
Index: sysstat/sysstat.sysconfig.in
===================================================================
--- sysstat.orig/sysstat.sysconfig.in	2011-01-19 21:14:31.000000000 +0100
+++ sysstat/sysstat.sysconfig.in	2011-01-19 21:23:02.000000000 +0100
@@ -1,6 +1,7 @@
-# @PACKAGE_NAME@-@PACKAGE_VERSION@ configuration file.
+# @PACKAGE_NAME@ configuration file.
 
 # How long to keep log files (in days).
+# Used by sa2(8) script
 # If value is greater than 28, then log files are kept in
 # multiple directories, one for each month.
 HISTORY=@HISTORY@
Index: sysstat/cron/crontab.sample
===================================================================
--- sysstat.orig/cron/crontab.sample	2011-01-19 21:23:20.000000000 +0100
+++ sysstat/cron/crontab.sample	2011-01-19 21:23:35.000000000 +0100
@@ -1,7 +1,4 @@
 # Crontab sample for root or adm
-# Please update this crontab with the proper location
-# for sa1 and sa2 shell scripts (replace @SA_LIB_DIR@ with
-# /usr/lib/sa for example).
 #
 # 8am-7pm activity reports every 20 minutes during weekdays.
 # 0 8-18 * * 1-5 @SA_LIB_DIR@/sa1 1200 3 &
