Fix a warning given by gcc -Wformat.

Index: sysstat/rndr_stats.c
===================================================================
--- sysstat.orig/rndr_stats.c	2011-08-28 19:45:11.000000000 +0200
+++ sysstat/rndr_stats.c	2011-08-28 19:45:25.000000000 +0200
@@ -133,7 +133,7 @@ static void render(int isdb, char *pre,
 			}
 		}
 		else {
-			printf(txt[isdb]);	/* No args */
+			printf("%s", txt[isdb]);	/* No args */
 		}
 	}
 
